### ISSUE TITLE
Question About Data Augmentation

### ISSUE THREAD

**Issue by Yushu-Li** [kdiAAA/TDA#1]
Hello,

Firstly, I wanted to express my gratitude for your outstanding paper and for generously sharing your code.

Upon reviewing the TDA experiments on the OOD benchmark (A/V/R/S), I noticed that they run rather slowly. My understanding was that AugMix augmentation is employed with 63 views for each test image in the OOD datasets. However, it was mentioned in the paper that only original test images are utilized.

In the "utils.py" file, the "build_test_data_loader" function utilizes the "get_ood_preprocess" function for OOD datasets.
`aug_preprocess = AugMixAugmenter(base_transform, preprocess, n_views=63, augmix=True)`
This command appears to generate 63 views of augmentation for each individual test image.

I'd like to confirm if I've misunderstood something related to this part.
Thank you very much.

Best,
Yushu

**Comment by kdiAAA** [comment]
Hello Yushu,

Thank you for your interest in our work! For the OOD benchmark datasets (A/V/R/S), we used the same augmentations as described in TPT. We did not detail this in the paper as it’s not a primary contribution. Also, the paper does not specify that only original test images are used; the augmentations seen in "utils.py" are indeed part of our intended approach.

Hope this helps clarify your queries!

Best,
Adilbek
